src/renderer/fonts/caching_shaper.rs

```diff
@@ -311,3 +312,18 @@ impl CachingShaper {
         self.blob_cache.get(&key).unwrap()
     }
 }
+
+fn points_to_pixels(value: f32) -> f32 {
+    // Fonts in neovim are using points, not pixels.
+    //
+    // Skia docs is incorrectly stating it uses points, but uses pixels:
+    // https://api.skia.org/classSkFont.html#a7e28a156a517d01bc608c14c761346bf
+    // https://github.com/mono/SkiaSharp/issues/1147#issuecomment-587421201
+    //
+    // So, we need to convert points to pixels.
+    let pixels_per_inch = 96.0;
```

Comment:
This feels suspicious to me. Pixels per inch are dependent on the monitor dpi settings aren't they?

Reply:
I think yes, but here we only care about converting pt to px, so it's unrelated. Scaling according to DPI happens later, when we multiply by scale_factor from winit.

I'll add prefix 'default' here to avoid confision, and small note.

Reply:
Probably better only comment, prefix would make variable names too lonk.